optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -444,6 +467,7 @@ def custom_forward(*inputs):
                     token_idx=token_idx,
                     attn_softmax_bf16=attn_softmax_bf16,
                     reuse_cache=reuse_cache,
+                    use_flash_attention=use_flash_attention,
```

Comment:
Also pass the flash attention parameter in line 452 of create_custom_forward(module).

Reply:
Added in second commit. Also did alignment to #583 .
cc @mandy-li